content/en/logs/guide/logs-rbac.md

```diff
@@ -2,132 +2,95 @@
-
-{{< img src="logs/guide/rbac/app-api_keys.png" alt="Check API and APP Keys" style="width:60%;">}}
+If you plan to use the Datadog API, you need an API key and an app key from an Admin user. API keys and app keys can be generated on your [Datadog account API key page][9]. Ensure the app key is associated with a user who has the necessary permissions. For more information, see [API and app keys][10].
 
-Throughout this guide, you will need to replace all occurrences of `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key, respectively. This guide also assumes that you have a terminal with `CURL`.
+In this guide, replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and application key, respectively. A terminal with `CURL` is also required.
 
-
-### Get permission IDs
+### Obtaining permission IDs
 
 **Note**: This section is only required if you intend on using the Datadog API to set up RBAC.
 
-Use the [Permissions API][11] to get the list of all existing permissions. The answer is an array of permissions such as the one below (the `logs_read_data` permission has the `<PERMISSION_ID>` `1af86ce4-7823-11ea-93dc-d7cad1b1c6cb`, which is all you need to know about that permission).
+If you plan to use the Datadog API, use the [Permissions API][11] to get all existing permissions. You need the permission ID for actions such as granting roles specific permissions. **Note**: The permission IDs change depending on your selected Datadog site ({{< region-param key="dd_site_name" >}}).
 
 ```bash
 curl -X GET "https://app.datadoghq.com/api/v2/permissions" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>"
 ```
 
-```json
-[...]
-{
-    "type": "permissions",
-    "id": "1af86ce4-7823-11ea-93dc-d7cad1b1c6cb",
-    "attributes": {
-        "name": "logs_read_data",
-        "display_name": "Logs Read Data",
-        [...]
-    }
-}
-[...]
-```
+## Setting up roles
 
-**Note**: The permission IDs change depending on the Datadog site (Datadog US, Datadog EU, etc.) you are using.
+This section guides you through creating two roles, `ACME Admin` and `ACME User`, granting them basic log permissions, and assigning users to these roles.
 
-## Set up roles
-This section explains how to create two roles, `ACME Admin` and `ACME User`; how to grant both roles minimal log permissions (extended later on in this guide); and how to assign users either role.
 
-### Create a role
+### Creating a role
 
 {{< tabs >}}
 {{% tab "UI" %}}
 
-In the [Groups Section][1] of Datadog Organization Settings, use the Add Role button within the Role tab to create the new `ACME Admin` and `ACME User`roles.
-
-{{< img src="logs/guide/rbac/add_role.png" alt="Add a new role" style="width:60%;">}}
+1. Navigate to the [Groups Section][1] in Datadog Organization Settings.
```

Comment:
```suggestion
1. Navigate to the [Roles section][1] in the Datadog Organization Settings.
```

Reply:
I don't see any reference to `Groups` on the linked page -- not sure if the UI may have been updated since this content was written